File: Flying Car/Flying Car/FlyingCar.m
//
//  FlyingCar.m
//  Flying Car
//
//  Created by Pratikbhai Patel on 3/27/15.
//  Copyright (c) 2015 Pratikbhai Patel. All rights reserved.
//

#import "FlyingCar.h"

@implementation FlyingCar

- (instancetype)init
{
    self = [super init];
    if (self) {
        self.canFly = YES;
        self.isConvertible = YES;
        self.isAutomatic = YES;
        self.altitude = 3000;
    }
    return self;
}

@end

File: Flying Car - Swift/Flying Car - Swift/FlyingCar.swift
//
//  FlyingCar.swift
//  Flying Car - Swift
//
//  Created by Pratikbhai Patel on 3/27/15.
//  Copyright (c) 2015 Pratikbhai Patel. All rights reserved.
//

import Foundation

class FlyingCar {
    var canFly:Bool?
    var isConvertible:Bool?
    var isAutomatic:Bool?
    var altitude:Int?
    
    init() {
        canFly = false
        isConvertible = false
        isAutomatic = true
        altitude = 3000
    }
}